mixer/adapter/kubernetesenv/kubernetesenv.go

```diff
@@ -183,7 +185,13 @@ func (h *handler) GenerateKubernetesAttributes(ctx context.Context, inst *ktmpl.
 	return out, nil
 }
 
-func (h *handler) Close() error {
+func (h *handler) Close() (err error) {
+	defer func() {
+		if recover() != nil {
```

Comment:
sorry i am not familiar with this code. Is recover() defined somewhere?

Reply:
It is part of golang: https://blog.golang.org/defer-panic-and-recover